src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Mvc/Views/Tournaments/Details.cshtml

```diff
@@ -11,8 +11,8 @@
 
 <script src="@Url.Content("~/Scripts/jquery.unobtrusive-ajax.js")" type="text/javascript"></script>
 <script>
-    var onArchiveSuccess = function() {
-        alert('Tournament was successfully archived');
+    var onSuccess = function () {
+        alert('@Model.ChangingStateMessage'.concat(' now.'));
```

Comment:
So why put "now" here?

Reply:
Because it's being shown right after archiving. So when you press "Archive Tournament" is shows "Tournament {name} is archived now."

Reply:
To show it as a result of an action.